src/ZTUtils/Zope.py

```diff
@@ -298,10 +298,6 @@ def simple_marshal(v):
     if isinstance(v, bytes):
         # Py 3 only
```

Comment:
```suggestion
```

This line is also no longer needed.

Reply:
Well spotted. I have removed the comment.